src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -34,6 +33,7 @@
     private final UserService userService;
     private final EventProducer eventProducer;
     private final CaseRepository caseRepository;
+    private final MoreTimeRequestRule moreTimeRequestRule;
 
     @Autowired
     public ClaimService(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00107)